contributing.md

```diff
@@ -124,3 +124,15 @@ content
 To get a sense of how content is transformed, see [crowdin.yml](crowdin.yml)
 
 [Crowdin]: https://crowdin.com/project/electron
+
+### Limited Proofreader Role
+
+There are two options to configure the proofreader role on crowdin.
+(You can change the role of translators in Translators tab on the project setting)
+
+- General proofreader: translate, approve strings for all languages.
+![general_proofreader](https://user-images.githubusercontent.com/8563047/38429142-8b8dc10e-39f8-11e8-9278-14314b557ca3.png)
+- The proofreader for languages: give a proofreader role for specific language
+![limited_proofreader](https://user-images.githubusercontent.com/8563047/38429156-96222196-39f8-11e8-9f1a-9ce012ecfa1d.png)
```

Comment:
Can you show in the screenshot or explain in words how this languages sidebar is opened?

Reply:
Good idea 👍  It would be better to attach gif.